packages/edit-site/src/components/global-styles/hooks.js

```diff
@@ -99,15 +99,35 @@ export function useSupportedStyles( name, element ) {
 	return supportedPanels;
 }
 
-export function useUniqueTypographyVariations() {
+export function useColorVariations() {
+	const colorVariations = useCurrentMergeThemeStyleVariationsWithUserConfig( {
+		property: 'color',
+	} );
+	/*
+	 * Filter out variations with no settings or styles.
+	 */
+	return colorVariations?.length
+		? colorVariations.filter( ( variation ) => {
+				const { settings, styles, title } = variation;
+				return (
+					title === 'Default' || // Always preseve the default variation.
```

Comment:
We probably need this to be translateable

Reply:
Done in [3a0eb63](https://github.com/WordPress/gutenberg/pull/59717/commits/3a0eb635446cb944102a74fdef2c3db7fdb6184c)